config/examples/Creality/CR-10S/CrealityV1/Configuration.h

```diff
@@ -2356,7 +2356,7 @@
 #define PREHEAT_1_FAN_SPEED     0 // Value from 0 to 255
 
 #define PREHEAT_2_LABEL       "ABS"
-#define PREHEAT_2_TEMP_HOTEND 250
+#define PREHEAT_2_TEMP_HOTEND 210 // ABS working temp between 210 and 250 C.
```

Comment:
I’d recommend dropping this to 235 to pass the max temperature check instead of dropping it to 210.

Reply:
Sounds good to me.

Reply:
210 + 15 = 225 … so it can be reduced by 10°C if preferred.